web/src/main/java/org/openmrs/web/filter/StartupFilter.java

```diff
@@ -175,9 +175,9 @@ private void initializeVelocity() {
 			velocityEngine = new VelocityEngine();
 			
 			Properties props = new Properties();
-			props.setProperty(RuntimeConstants.RUNTIME_LOG, "startup_wizard_vel.log");
```

Comment:
Is this required by the ticket?

Reply:
yes because the new version of apache velocity 2.1 has no RUNTIME_LOG and RUNTIME_LOG_LOGSYSTEM_CLASS in its API, so i had to look for the equivalent in the new versioned API

Reply:
https://velocity.apache.org/engine/2.0/apidocs/org/apache/velocity/runtime/RuntimeConstants.html